resume.md

```diff
@@ -1,49 +1,57 @@
-# FIRST LAST NAME
+# Jacqueline Li
+(Canadian citizen with F1 Visa)
 
-(123) 456 - 7890 | email@gmail.com
-
-www.website.com | github.com/myusername | linkedin.com/myusername
+(123) 456 - 7890 | email@gmail.com | linkedin.com/myusername | github.com/myusername
 
 ## EDUCATION
-**University of College** | B.S. Degree ... *MONTH 20XX*
-* **Coursework**: Data Structures, Algorithms, Upper Division CS Elective, Math Course, Statistics, Data Science, ETC.
+**University of California, Berkeley** | BA in Computer Science, Expected Graduation *DEC 2020*
+* **Coursework**:
+Visualization for Design
+The Structure and Interpretation of Computer Programs
+Data Structures
+Information Devices and Systems
+Discrete Mathematics and Probability Theory
+Computer Architecture
+Introduction to Artificial Intelligence
+Computer Security
+Computational Models of Cognition
+Operating Systems and System Programming
+Efficient Algorithms and Intractable Problems
 
 ## SKILLS
-* **Languages**: Python, Java, C, SQL, ...
-* **Libraries/Frameworks**: Pandas, TensorFlow, PyTorch, ...
-* **Version Control**: Git, Bitbucket, Sourcetree, GithubDesktop
-* *...*
+* **Languages**: Python ■ Java ■ C ■ SQL ■ LaTeX ■ Go ■ RISC-V ■ Scheme ■ LISP
```

Comment:
include any interesting libraries you've used odds are you have used
* JUnit, Numpy, Math

Maybe consider a general technologies bucket
* **Technologies**: Git, Blah blah